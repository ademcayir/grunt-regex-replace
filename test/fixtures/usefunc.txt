Keep [skipped]. Expect [replaced]. Some text.
