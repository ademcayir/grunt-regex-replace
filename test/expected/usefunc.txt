Keep [skipped]. Expect [changed]. Changed text.
